components/system_notice/notices.jsx

```diff
@@ -33,6 +35,13 @@ export default [
                 defaultMessage='If you’ve created or installed integrations in the last two years, find out how <a href="https://about.mattermost.com/default-apiv3-deprecation-guide" target="_blank">recent changes</a> may have affected them.'
             />
         ),
+        cantForget: false,
+        show: (serverVersion, config) => {
+            if (config.InstallationDate < new Date(2018, 6, 16, 0, 0, 0, 0).getTime()) {
```

Comment:
nit: was thrown off for a bit with semantics of this `show` vs. the one below, as we default to `return false` here but `return true` there, and so the checks are all inverted.

Reply:
Good point, i going to change it.

Reply:
Done! :)